### PR TITLE
[REFACTOR] People In Course

### PR DESCRIPTION
**Pull request by IlmaMannix**
Change some id name and add some test

### CODE REVIEW

**Review by AldiNFitrah — APPROVED** [review]
was scared to review this due to the large number of file changes. Turns out they're all just coverage lol. LGTM!

**Review by fxdros — COMMENTED** [review]
I think it will be better if we resolve the conflict before merging

File: src/resources/app/pages/PeopleInCourse.vue [hunk 1]
```diff
@@ -1,61 +1,69 @@
...
+          </v-row>
+        </v-col>
+      </div>
+    </v-container>
+  </v-app>
 </template>
 <script>
 import api from "../../api/people";
 
 export default {
```
Comment (Zsaschz) [inline src/resources/app/pages/PeopleInCourse.vue:38]: please use AppLayout.vue and courseStream.vue

**Review by kanziebub — APPROVED** [review]
LGTM, don't forget to resolve conflicts!

**Pull request closed without merging**